pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -429,6 +429,9 @@ func (r *Builder) mapResources(vm *model.Workload, object *cnv.VirtualMachineSpe
 			object.Template.Spec.Domain.CPU.IsolateEmulatorThread = true
 		}
 	}
+	if object.Instancetype != nil {
+		return
```

Comment:
I think early return doesn't really fit here - it's easy to miss this and add something at the bottom of this function, I'd rather prefer to have a block that runs only when the instance type is not set instead and still get to the bottom of this function